cli/lib/kontena_cli.rb

```diff
@@ -87,6 +109,20 @@ def self.cli_root(*joinables)
       File.join(Kontena.root, 'lib/kontena/cli', *joinables)
     end
   end
+
+  def self.logger
+    return @logger if @logger
+    require 'logger'
+    @logger = Logger.new($LOG_TARGET, 'weekly')
+    @logger.level = ENV["DEBUG"] ? Logger::DEBUG : Logger::INFO
+    @logger.progname = 'CLI'
```

Comment:
The agent/server have the nice touch of having this `def self.logger` being an `include` module, that uses the `self.class.name` as the progname. Would that also make sense for the CLI?